tests/test-cases/Group3-Docker-Compose/3-04-Docker-Compose-Fixtures.robot

```diff
@@ -0,0 +1,419 @@
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/invalid-composefile/invalid.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Links composefile
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/links-composefile/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Logging composefile legacy
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/logging-composefile-legacy/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Logging composefile
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/logging-composefile/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Logs composefile
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/logs-composefile/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Logs tail composefile
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/logs-tail-composefile/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Longer filename composefile
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/longer-filename-composefile/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Multiple composefiles
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/multiple-composefiles/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Net container
+    Set Environment Variable  COMPOSE_HTTP_TIMEOUT  300
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f %{GOPATH}/src/github.com/vmware/vic/tests/compose/fixtures/net-container/docker-compose.yml up -d
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+Networks
```

Comment:
If you are using the `external network` syntax here, make sure the network is created before compose-up